basic/src/lib.rs

```diff
@@ -2,6 +2,7 @@
 #![allow(unused)]
 
 extern crate alloc;
+extern crate std;
```

Comment:
leftover from debugging?

Reply:
Indeed, good catch. (Why does the tooling not flag this automatically?)